web/polymer_elements/paper_toolbar/adding_button_rows/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <link rel="import" href="packages/polymer_elements/paper_toolbar.html">
```

Comment:
I think I would still prefer dart imports in the script tag below, for the same canonicalization reasons.